rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -208,6 +209,35 @@ local function route_to_alternative_balancer(balancer)
     return false
   end
 
+  if traffic_shaping_policy.canaryConsistency and traffic_shaping_policy.canaryConsistency ~= "" then
+    local seed = traffic_shaping_policy.hashSeed
+    if not seed then
+      seed = ""
+    end
+    if traffic_shaping_policy.canaryConsistency == "header" then
```

Comment:
How about add a blank line here? the code is more readable

Reply:
sure